packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -215,6 +203,36 @@ export default class InsertTableView extends View {
 		} );
 	}
 
+	/**
+	 * Creates a new Button for the grid.
+	 *
+	 * @private
+	 * @returns {module:ui/button/buttonview~ButtonView}
```

Comment:
Missing docs `@param` for `locale`, `row`, `column`, and `label`.

Reply:
filled